neo/Consensus/ConsensusContext.cs

```diff
@@ -72,9 +73,16 @@ public void Deserialize(BinaryReader reader)
             PrevHash = reader.ReadSerializable<UInt256>();
             BlockIndex = reader.ReadUInt32();
             ViewNumber = reader.ReadByte();
-            Validators = reader.ReadSerializableArray<ECPoint>();
+            Validators = reader.ReadSerializableArray<ECPoint>(byte.MaxValue);
```

Comment:
It is not necessary to check the number of items here, because the serialization and deserialization of the `ConsensusContext` are performed by the local node and will not accept the data passed by the external nodes, so there is no security problem.

If you really want to check, you can define a global constant `MaxValidatorsCount`.

Reply:
Might as well have it; there are places where we use it in the `RecoveryMessage`.

Reply:
Then it should be added to `RecoveryMessage.Deserialize()`.

Reply:
Where do you think the constant should live? Inside of `ConsensusContext`? Or should we create a new class for such constants?

Reply:
`ConsensusService` is a good place, I think.

Reply:
I think the constant should live in a place more general than inside the `RecoveryMessage`; yes I will use it in `RecoveryMessage.Deserialize()` once we decide where to put it.

Reply:
Done.